catalog/package.json

```diff
@@ -44,11 +44,7 @@
     "test:watch": "npm run test:only -- --watch",
     "prettify": "prettier --write"
   },
-  "browserslist": [
-    "last 2 versions",
-    "> 1%",
-    "IE 10"
-  ],
+  "browserslist": ["last 2 versions", "> 1%"],
```

Comment:
calling `npm install` formats this array to multiline:

```diff
-  "browserslist": ["last 2 versions", "> 1%"],
+  "browserslist": [
+    "last 2 versions",
+    "> 1%"
+  ],
```

Linux, node v15.5.0, npm 7.3.0

Reply:
```suggestion
  "browserslist": [
    "last 2 versions",
    "> 1%"
  ],
```